content/usgs-packages/sbtools_Intro.Rmd

```diff
@@ -0,0 +1,56 @@
+---
+title: "sbtools - Introduction"
+date: "9999-08-01"
+author: "Lindsay R. Carr"
+slug: "sbtools-intro"
+image: "img/main/intro-icons-300px/r-logo.png"
+output: USGSmarkdowntemplates::hugoTraining
+parent: Introduction to USGS R Packages
+weight: 2
+draft: true
+---
+
+```{r setup, include=FALSE, warning=FALSE, message=FALSE}
+library(knitr)
+
+knit_hooks$set(plot=function(x, options) {
+  sprintf("<img src='../%s%s-%d.%s'/ title='%s'/>", 
+          options$fig.path, options$label, options$fig.cur, options$fig.ext, options$fig.cap)
+
+})
+
+opts_chunk$set(
+  echo=TRUE,
+  fig.path="static/sbtools-intro/",
+  fig.width = 6,
+  fig.height = 6,
+  fig.cap = "TODO"
+)
+
+set.seed(1)
+```
+
+## Lesson Summary
+
+`sbtools` is an R package that enables access to the USGS web storage platform, [ScienceBase](https://www.sciencebase.gov). In this lesson, will teach how to interact with ScienceBase from the R console in order to create reproducible workflows.
+
+ScienceBase is collaborative scientific data and information management platform used by USGS scientists and their collaborators. Account holders can upload data using standard metadata practices, and choose to share the content publically or keep it private. An account is not needed to download publically available data, so the platform is a great way to share data released with publications.
```

Comment:
i would take out "using standard metadata practices" - you can upload any old data you want without adhering to any standards. then maybe add a link to their data release page toward the end of this paragraph: https://www.sciencebase.gov/about/content/data-release